novawallet/Common/Model/ChainRegistry/LocalChain/ChainModel.swift

```diff
@@ -533,3 +555,75 @@ extension ChainModel {
         return Set(priceIds)
     }
 }
+
+extension ChainModel {
+    static func partialCustomChainModel(
```

Comment:
Do we still need this if we are using separate model for partial chain?

Reply:
No, we don't. Forgot to cut it out. Thanks!